### PR TITLE
CSV file parsing implementation

### PR DESCRIPTION
**Pull request by julia-shvabovich**

### CODE REVIEW

**Review by aleksandr-hrankin — APPROVED** [review]

File: src/main/resources/wrong_format.csv [hunk 1]
```diff
@@ -0,0 +1 @@
+number,B001E4KFG0,A3SGXH7AUHU8GW,delmartian,1,1,5,1303862400,Good Quality Dog Food,5
```
Comment (RomanDubovskyi) [inline src/main/resources/wrong_format.csv]: empty line in the end of the file

File: src/test/java/amazon/review/CsvParserServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        expectedDto.setHelpfulnessNumerator(1);
+        expectedDto.setHelpfulnessDenominator(1);
+        expectedDto.setScore(5);
+        expectedDto.setTime(1303862400L);
+        expectedDto.setSummary("Good Quality Dog Food");
+        expectedDto.setText("I have bought several of the Vitality canned dog food products"
+                + " and have found them all to be of good quality.");
+        String line = readerService.read(FILE_NAME).get(0);
+        Assert.assertEquals(expectedDto,
+                new CsvParserServiceImpl().parseLine(line));
```
Comment (RomanDubovskyi) [inline src/test/java/amazon/review/CsvParserServiceImplTest.java]: declare CsvParserServiceImpl as variable

File: src/main/resources/wrong_format.csv (hunk 1 above)
Comment (julia-shvabovich) [reply src/main/resources/wrong_format.csv]: done :)

File: src/test/java/amazon/review/CsvParserServiceImplTest.java (hunk 2 above)
Comment (julia-shvabovich) [reply src/test/java/amazon/review/CsvParserServiceImplTest.java]: done

File: src/test/java/amazon/review/CsvParserServiceImplTest.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        expectedDto.setText("I have bought several of the Vitality canned dog food products"
+                + " and have found them all to be of good quality.");
+        String line = readerService.read(FILE_NAME).get(0);
+        Assert.assertEquals(expectedDto, csvParserService.parseLine(line));
+    }
+
+    @Test
+    public void parseWrongFormatLine() {
+        assertThrows(RuntimeException.class, () -> {
+            new CsvParserServiceImpl().parseLine(readerService.read(WRONG_FORMAT).get(0));
```
Comment (RomanDubovskyi) [inline src/test/java/amazon/review/CsvParserServiceImplTest.java]:
```suggestion
            csvParserService.parseLine(readerService.read(WRONG_FORMAT).get(0));
```

File: src/test/java/amazon/review/CsvParserServiceImplTest.java [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        });
+    }
+
+    @Test
+    public void parseEmptyLine() {
+        assertThrows(RuntimeException.class, () -> {
+            new CsvParserServiceImpl().parseLine(null);
```
Comment (RomanDubovskyi) [inline src/test/java/amazon/review/CsvParserServiceImplTest.java]: same here

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**